### DISCUSSION TITLE
Update Angular 16

### DISCUSSION THREAD

**Discussion by lrm6666** [ShankyTiwari/ng-material-multilevel-menu#220] [General]
Hi,
When will you release the compatible version with angular 16?
Thanx in advance

**Comment by ShankyTiwari** [comment]
In the next 2 weeks for sure, really sorry for the delay. Thanks

**Reply by lrm6666** [reply]
Is 6.0.0 (an then 6.0.2) the compatible version with Angular 16?
Thanx for your work!

**Comment by ShankyTiwari** [comment]
Yes, it is compatible. Please use `6.0.2` version. Thanks
